paperclip-permanent_records.gemspec

```diff
@@ -27,7 +27,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'permanent_records', '>= 5.0'
 
   spec.add_development_dependency 'appraisal', '~> 2.5'
-  spec.add_development_dependency 'bundler', '~> 2.4'
+  spec.add_development_dependency 'bundler', '~> 2'
```

Comment:
Any specific reason for changing this? GH Actions?

Reply:
Yes, that is the default version that comes with ruby but I just saw that we could configure 2.4 for the action.